l10n_uy_reports/static/src/components/uruguayan_report/filters.js

```diff
@@ -0,0 +1,27 @@
+import { _t } from "@web/core/l10n/translation";
+import { patch } from "@web/core/utils/patch";
+import { AccountReportFilters } from "@account_reports/components/account_report/filters/filters";
+
+patch(AccountReportFilters.prototype, {
+    get selectedTaxType() {
+        const availableTypes = Object.keys(this.controller.options.uy_vat_book_tax_types_available);
+        const selectedTypes = Object.values(
+            this.controller.options.uy_vat_book_tax_types_available,
+        ).filter((type) => type.selected);
+
+        if (selectedTypes.length === availableTypes.length || selectedTypes.length === 0) {
+            return _t("All");
```

Comment:
```suggestion
            return "All";
```
quizás habría que hacer esto?